.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: "CI"
+
+env:
+  DOTENV_CONFIG_PATH: "./.env.example"
+
+on:
+  push:
```

Comment:
We shouldn't need to run on push by default the pull request event runs on open synchronized and reopened

Reply:
We'll address this in VEN-2091